cmd/instance_pool.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+var instancePoolCmd = &cobra.Command{
+	Use:   "instance-pool",
+	Short: "Instance pool managment",
+	Long: `The Instance Pool is a managed service allowing users
+to provision a group of similar Compute instances created,
+ensuring that at any moment there is a user-specified
+fixed number of instances running.`,
+}
+
+func init() {
+	RootCmd.AddCommand(instancePoolCmd)
+}
+
+func getInstancePool(id, zone *egoscale.UUID) (*egoscale.InstancePool, error) {
```

Comment:
Given that you have implemented a `getInstancePoolByName()` function it'd be more logical to name this one `getInstancePoolByID()`.

Reply:
fixed